dialogue-core/src/main/java/com/palantir/dialogue/core/UserAgentEndpointChannel.java

```diff
@@ -60,7 +61,7 @@ public ListenableFuture<Response> execute(Request request) {
     }
 
     private static UserAgent augmentUserAgent(UserAgent baseAgent, Endpoint endpoint) {
-        return tryAddEndpointAgent(baseAgent, endpoint).addAgent(DIALOGUE_AGENT);
+        return tryAddEndpointAgent(baseAgent, endpoint).addAgent(DIALOGUE_AGENT).addAgent(JDK_AGENT);
```

Comment:
I'm considering exposing something like `UserAgent addAgents(Iterable<Agent>)` to `conjure-java-runtime-api` to avoid multiple immutable copies, but that can be a follow on PR

Reply:
https://github.com/palantir/conjure-java-runtime-api/pull/902